fletcher/base.py

```diff
@@ -190,6 +202,21 @@ def dtype(self):
         # type: () -> ExtensionDtype
         return self._dtype
 
+    # reduction ops #
+    def _reduce(
+        self, name, axis=0, skipna=True, numeric_only=None, min_count=None, **kwargs
+    ):
+        func = getattr(self, name, None)
+        if func is None:
```

Comment:
Could you emit a Warning when we take a numpy function here? NumPy cannot really deal with nulls thus this route is ok but discouraged.